### PR TITLE
[ExpressionLanguage] Improve test coverage

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php [hunk 1]
```diff
@@ -287,4 +287,11 @@ function (ExpressionLanguage $el) {
             ],
         ];
     }
+
+    public function testParseAlreadyParsedExceptionReturnsSameObject()
```
Comment (OskarStark) [inline src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php]: Exception?

Comment (alexandre-daubois) [reply src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php]: Typo from me, I meant to say "Expression" actually... thanks!

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/ExpressionLanguage/Tests/Node/BinaryNodeTest.php [hunk 2]
```diff
@@ -207,4 +207,24 @@ public function testCompileMatchesWithInvalidRegexpAsExpression()
         $node->compile($compiler);
         eval('$regexp = "this is not a regexp"; '.$compiler->getSource().';');
     }
+
+    public function testDivisionsByZero()
```
Comment (xabbuh) [inline src/Symfony/Component/ExpressionLanguage/Tests/Node/BinaryNodeTest.php]:
```suggestion
    public function testDivisionByZero()
```

**Review by xabbuh — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @alexandre-daubois.

**Pull request merged**